.githooks/pre-commit

```diff
@@ -101,4 +101,18 @@ then
   done
 fi
 
+# Check Apache License header for source files.
+if [ $NUM_TOTAL_FILES_CHANGED -gt 0 ]
+then
+  echo_status "Checking Apache License Header ..."
+  go get -u github.com/google/addlicense
+  addlicense -c "Google LLC" -l apache .
```

Comment:
Could you please confirm that this doesn't actually add the headers, just does a check?

Reply:
yes, `-check` enables the check mode. Thanks